### DISCUSSION TITLE
How do I make a colonist skin only apply to my colonists?

### DISCUSSION THREAD

**Discussion by anameeguy** [ldtteam/minecolonies#9814] [Q&A]
If I were to make a skin for my colonists could I make it so that it only shows on mine? That would mean that I would not need to turn it off whenever I go to a different colony as well as making wars less confusing. Even if I have to get a client side mod I will be happy.

**Comment by Talyda** [comment]
If you join patreon at the correct tier you can do that! The $10(US) per month tier (with the patreon skins at least!)

**Reply by anameeguy** [reply]
I want to make my own.

**Reply by SkullGamer205** [reply]
> I want to make my own.

Not only you, the-aNamee.

**Reply by Talyda** [reply]
> > I want to make my own.
> 
> Not only you, the-aNamee.

You can make your own SkullGamer. Easier to search discord, or ask there so we can link you things

**Reply by SkullGamer205** [reply]
> > > I want to make my own.
> > 
> > 
> > Not only you, the-aNamee.
> 
> You can make your own SkullGamer. Easier to search discord, or ask there so we can link you things

Yep. But...
How can I change Citizen Style & Name pack if I need to subscribe to patreon?
Can I apply different styles for only me or my friends?

**Reply by uecasm** [reply]
For free (other than the time investment to create one), you can apply a single custom style to all colonies on the server.

If you want to apply different styles to different colonies, then the owner of each colony must be a patreon (or have sufficient lifetime contribution), as Talyda already said.

**Reply by Raycoms** [reply]
We pay people that make complete styles we like and would include in the mod and you get also feature access for making one we included in the mod.

**Reply by Imperec-rus** [reply]
So, anyway. How can I unlock these features without a Patreon? It's just unfair. I want to create my own style, but I have to pay for it. Moreover, this violates Mojang's rules. Modifications should not be paid for. I respect creativity, but still...

**Reply by MotionlessTrain** [reply]
You can make a resource pack to change the default citizen styles with the ones you make

**Reply by Talyda** [reply]
And it doesn't violate Mojang's rules BTW

**Reply by Raycoms** [reply]
First, mojang explicitly allows in their EULA creators to have purely cosmetic things locked behind payment. You're only not allowed to sell full mods.

I think you're just being incredibly greedy and even willing to harm the development team that puts countless hours into this just so that you can get even more free stuff.

**Reply by Imperec-rus** [reply]
I would calmly support the developers on Patreon if it weren't for my regional position. That's why I'm interested in how to get cosmetics for people like me. So is there still a way to unlock without creating texture packs?

**Reply by Raycoms** [reply]
Either paypal/patreon, or resource pack with your own style. Or submit your own style for inclusion in the mod and you could get it also unlocked.

**Reply by Imperec-rus** [reply]
1. I can't create a PayPal account. I'm from another country.
2. If I send my style, not everyone will like it. It was created for a specific mod build...
3. The texture pack applies to all colonists, not certain ones. The advice is depressing.

**Reply by Raycoms** [reply]
We cannot have everything in live.

**Reply by Imperec-rus** [reply]
But still. How do I enable these functions?

**Reply by Raycoms** [reply]
You don't.

**Reply by Imperec-rus** [reply]
Cosmetics should not cost 10$. Lmao

**Reply by Raycoms** [reply]
Cosmetics don't cost 10$, cosmetics are a reward for people that want to support the development of the mod given that a team of people puts upwards to 160 hours a month into this mod.

**Reply by anameeguy** [reply]
Could everybody please not use this as a place to complain?
